numba/npyufunc/gufunc_scheduler.h

```diff
@@ -1,8 +1,26 @@
 #ifndef GUFUNC_SCHEDULER
 #define GUFUNC_SCHEDULER
 
-#include <stdint.h>
-#if __SIZEOF_POINTER__ == 8
+/* define int64_t and uint64_t for VC9 */
```

Comment:
this comment is slightly inaccurate.  The #ifdef will branch will be taken for all versions of Visual Studio, not just VS2008